ddl/label/rule.go

```diff
@@ -43,10 +53,11 @@ var (
 
 // Rule is used to establish the relationship between labels and a key range.
 type Rule struct {
```

Comment:
I suggest you giving a better name than `Rule`.

Reply:
Any suggestion?

Reply:
Or maybe change `Rules` to `KeyRanges`?

Reply:
I have no idea about it for now.